packages/canary-integration/src/__tests__/model-instance-document.test.ts

```diff
@@ -2,36 +2,139 @@ import { jest } from '@jest/globals'
 import getPort from 'get-port'
 import { AnchorStatus, CeramicApi, CommitType, IpfsApi } from '@ceramicnetwork/common'
 import { createIPFS } from '@ceramicnetwork/ipfs-daemon'
-import { ModelInstanceDocument } from '@ceramicnetwork/stream-model-instance'
+import { ModelInstanceDocument, ModelInstanceDocumentMetadata } from '@ceramicnetwork/stream-model-instance'
 import { createCeramic } from '../create-ceramic.js'
 import { anchorUpdate } from '@ceramicnetwork/core/lib/state-management/__tests__/anchor-update'
 import { Ceramic } from '@ceramicnetwork/core'
 import { CeramicDaemon, DaemonConfig } from '@ceramicnetwork/cli'
 import { CeramicClient } from '@ceramicnetwork/http-client'
 import { StreamID } from '@ceramicnetwork/streamid'
 import first from 'it-first'
+import { Model, ModelAccountRelation, ModelDefinition } from '@ceramicnetwork/stream-model'
+
+const CONTENT_VALID_1 = {
```

Comment:
same comment here as I made below on `model-instance-document-handler.test.ts`.  Adding too much into one test obfuscates the core of what we're trying to get out of that test.  I think the tests at each layer should focus on testing what that layer is adding, while relying on tests of lower layers to test that lower-level functionality.  I think we can mostly put this test file back to the way it was and just have a simple test that schema enforcement is happening, while relying on the lower-level tests to cover all the edge cases of schema validation.